pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -920,7 +920,14 @@ self: super: {
   # Generate cli completions for dhall.
   dhall = generateOptparseApplicativeCompletion "dhall" super.dhall;
   dhall-json = generateOptparseApplicativeCompletions ["dhall-to-json" "dhall-to-yaml"] super.dhall-json;
-  dhall-nix = generateOptparseApplicativeCompletion "dhall-to-nix" (super.dhall-nix);
+  dhall-nix = generateOptparseApplicativeCompletion "dhall-to-nix" (
+    super.dhall-nix.override {
+      dhall = super.dhall_1_34_0.override {
+        repline = self.repline_0_4_0_0.override {
+          haskeline = self.haskeline_0_8_0_0;
+        };
+      };
```

Comment:
You could probably rewrite this a little cleaner with `overrideScope` (instead of a normal `override`). There should be a couple uses of it in this file to copy.

Reply:
Ok, done.